.github/workflows/lint-pr-title.yaml

```diff
@@ -14,7 +14,11 @@ jobs:
     name: conventional-commit
     runs-on: ubuntu-latest
     steps:
+<<<<<<< HEAD
```

Comment:
Seems have unresolved conflict

Reply:
Made the change locally but somehow didn't commit it before merging the merge commit :(